corehq/apps/app_manager/xpath.py

```diff
@@ -45,6 +46,9 @@ def interpolate_xpath(string, case_xpath=None):
         '#user': UserCaseXPath().case(),
         '#session/': session_var('', path=''),
     }
+    if module:
```

Comment:
Shouldn't this actually check whether fixture selection is active for the module: `module.fixture_select.active`?